x/tx/signing/context_test.go

```diff
@@ -97,6 +108,38 @@ func TestGetSigners(t *testing.T) {
 			}},
 			want: [][]byte{[]byte("foo"), []byte("bar")},
 		},
+		{
+			name: "nested repeated #2",
+			msg: &testpb.DeeplyNestedRepeatedSigner{
+				InnerOne: []*testpb.DeeplyNestedRepeatedSigner_InnerOne{
+					{
+						InnerTwo: []*testpb.DeeplyNestedRepeatedSigner_InnerOne_InnerTwo{
+							{
+								Signer: []string{hex.EncodeToString([]byte("foo")), hex.EncodeToString([]byte("bar"))},
+							},
+						},
+					},
+					{
+						InnerTwo: []*testpb.DeeplyNestedRepeatedSigner_InnerOne_InnerTwo{
+							{
+								Signer: []string{hex.EncodeToString([]byte("baz"))},
+							},
+						},
+					},
+					{
+						InnerTwo: []*testpb.DeeplyNestedRepeatedSigner_InnerOne_InnerTwo{
+							{
+								Signer: []string{hex.EncodeToString([]byte("qux")), hex.EncodeToString([]byte("fuz"))},
+							},
+							{
+								Signer: []string{hex.EncodeToString([]byte("bing")), hex.EncodeToString([]byte("bap"))},
+							},
+						},
+					},
+				},
+			},
+			want: [][]byte{[]byte("foo"), []byte("bar"), []byte("baz"), []byte("qux"), []byte("fuz"), []byte("bing"), []byte("bap")},
```

Comment:
Unrelated, I didn't know repeated signers were a thing. What happens if they are duplicated? Do we deduplicate them? I couldn't find it a test for it. Looks like to me that we don't do that.

Reply:
I don't see any support for deduplication.  Is this implementation aligned with the signing spec? I don't know where to find it.

Reply:
I don't know either tbh